docs/01-Using-Fleet/03-REST-API.md

```diff
@@ -6067,15 +6036,15 @@ Transforms a host name into a host id. For example, the Fleet UI use this endpoi
-| query                   | string  | query | Search query keywords. Searchable fields include `name`.                                                                                                                                                                                                                                                                                    |
-| team_id                 | integer | query | _Available in Fleet Premium_ Filters the software to only include the software installed on the hosts that are assigned to the specified team.                                                                                                                                                                                                   |
-| vulnerable              | bool    | query | If true or 1, only list software that has detected vulnerabilities                                                                                                                                                                                                                                                                          |
+| Name            | Type    | In    | Description                                                                                                                                    |
+| --------------- | ------- | ----- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| page            | integer | query | Allowed for compatibility with GET /api/v1/fleet/software but ignored                                                                          |
+| per_page        | integer | query | Allowed for compatibility with GET /api/v1/fleet/software but ignored                                                                          |
+| order_key       | string  | query | Allowed for compatibility with GET /api/v1/fleet/software but ignored                                                                          |
+| order_direction | string  | query | Allowed for compatibility with GET /api/v1/fleet/software but ignored                                                                          |
+| query           | string  | query | Search query keywords. Searchable fields include `name`.                                                                                       |
+| team_id         | integer | query | _Available in Fleet Premium_ Filters the software to only include the software installed on the hosts that are assigned to the specified team. |
+| vulnerable      | bool    | query | If true or 1, only list software that has detected vulnerabilities                                                                             |
```

Comment:
Curious, what did you run to sanitize the whole file?

Reply:
Ah, my emacs config does it automatically on save. It would be great to find a tool that everyone could use. Might be nice to do this in a separate commit though. I'll see if I can pick out the changes.